dts/bindings/fuel-gauge/sbs,default-sbs-gauge.yaml

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright 2023 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+compatible: "sbs,default-sbs-gauge"
+
+include: ["sbs,sbs-gauge-new-api.yaml", "battery_cutoff.yaml"]
+
+description: |
+  Default generic smart battery fuel gauge driver. Includes support for battery cutoff if enabled.
```

Comment:
I think nearly everywhere you use this it should be 'battery-cutoff'

Reply:
Perhaps you should just say 'cutoff' an then mention in one place that it means cutting off the battery?

Reply:
I'd like to keep it to battery cutoff to be more explicit and differentiating from charge cutoff.